app/graphql/cache.ts

```diff
@@ -61,6 +63,35 @@ const getWallets = ({
   return account.wallets
 }
 
+const getDisplayCurrencyAndFractionDigits = (cache: InMemoryCache) => {
+  // FIXME as RealtimePriceQuery is a singleton,
+  // could we have a way to not fetch DisplayCurrencyQuery?
+  const resCurrency = cache.readQuery<DisplayCurrencyQuery>({
+    query: DisplayCurrencyDocument,
+  })
+  const displayCurrency = resCurrency?.me?.defaultAccount?.displayCurrency
+
+  if (!displayCurrency) {
+    return { displayCurrency: null, fractionDigits: null }
+  }
+
+  const resCurrencyList = cache.readQuery<CurrencyListQuery>({
+    query: CurrencyListDocument,
+    variables: { currency: displayCurrency },
+  })
+  const currencyList = resCurrencyList?.currencyList
+
+  if (!currencyList) {
+    return { displayCurrency: null, fractionDigits: null }
+  }
+
+  const fractionDigits =
+    currencyList.find((currency) => currency.id === displayCurrency)?.fractionDigits ??
```

Comment:
If fraction digits is 0 here it will be falsy and then the value will be null.

Reply:
You can just remove the `?? null`

Reply:
not with ??:

```
% node
Welcome to Node.js v18.14.0.
Type ".help" for more information.
> 0 ?? "a"
0
> 0 || "a"
'a'
```

Reply:
I thought it behaved the same as `||`. My mistake.

Reply:
if I remove `?? null` then the return type of undefined. then I have to check both for `null` and `undefined` in the calling function, which I don't want to do